lua/lspconfig/server_configurations/leanls.lua

```diff
@@ -20,6 +24,19 @@ return {
         or util.find_git_ancestor(fname)
     end,
     on_new_config = function(config, root_dir)
+      local lake_version = ''
```

Comment:
We also need to check that a `lakefile.lean` exists.  (Otherwise `lake serve` will fail.)

Reply:
This works as-is for me like I said [here](https://github.com/Julian/lean.nvim/issues/229#issuecomment-1032565846) but let me know if I missed which case triggers this -- unless you mean we should fix single-file mode for `lake==3.0`, in which case yes that'd be nice but I don't have much energy to do it at the minute, maybe later.